addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -2442,3 +2442,16 @@ export function selectionMatchesInnerText(element) {
 export function isMacOS() {
     return window.navigator.userAgent.includes('Mac');
 }
+
+/**
+ * Removes all ZeroWidthSpace characters from a node (if text) or
+ * its descendants.
+ * It does not remove nodes.
+ *
+ * @param {Node} node
+ */
+export function removeZWSChars(node) {
```

Comment:
I suggest renaming this function to `removeZWS` to be consistent with `isZWS`.

Reply:
`isZWS` informs whether a **node** contains only a ZWS. Thus, is node "ZWS"? True or false.
`removeZWS` would make me think the function removes such node. But it does not, it removes ZeroWidthSpace characters in a node's content. The node is kept. It is meant to remove ZWS from nodes/elements that have ZWS and other stuff.
I agree that `removeZWS` is more consistent with that other function's name, but it could be deceiving.